components/ui/GoogleAdsense/index.tsx

```
@@ -1,4 +1,12 @@
-import { useEffect, useRef } from "react";
+import { useEffect, useRef, CSSProperties, JSX } from "react";
+import { AdStatus, GoogleAdsenseContainerProps } from "types/common";
+
+declare global {
+  // eslint-disable-next-line no-unused-vars
+  interface Window {
+    adsbygoogle: any[];
+  }
```

ESLint

[error] Do not declare interfaces outside of the /types directory. Centralize all interfaces in /types.

(no-restricted-syntax)